### ISSUE TITLE
The labels for the tests in Cats&Dogs are so wrong!

### ISSUE THREAD

**Issue by rizar** [mila-udem/fuel#322]
All the images from the test set are labelled 1 ("dog"). The offending line is [this one](https://github.com/mila-udem/fuel/blob/master/fuel/converters/dogs_vs_cats.py#L93). I used the links from the [quick start guide](https://ift6266h16.wordpress.com/class-project/getting-started/) to download the data, and test set images do not include the name of the animal in their filenames.

Should I download the data from elsewhere?

**Comment by bartvm** [comment]
My bad, the test set shouldn't have any labels. To get your test set performance you need to submit your predictions to Kaggle. Last year the rule was to use 0-20000 as your training set, 20000 to 25000 as your validation set, and Kaggle as your test set. I decided not to fix a validation set this year, in case people want to do stuff like cross-validation, but those faulty labels need to go... I'll also explain this more clearly on the project page.

**Comment by rizar** [comment]
Fixed by #324 and  #323

**Comment by DanlanChen** [comment]
Hello, why the test set should not have any labels?

**Comment by DanlanChen** [comment]
Then we should use the prediction class of  test set in fuel and test the accuracy in kaggle?

**Comment by bartvm** [comment]
Yes, you have your model create predictions for the images in the test set, and then upload those to the Kaggle website if you want to get your test set accuracy. You shouldn't have to do this very often, because model selection should be done with a validation set (which you should create yourself from the training set e.g. the last 5,000 examples).
